apps/electron-app/package.json

```
@@ -29,7 +29,9 @@
     "@electron-toolkit/preload": "^3.0.1",
     "@electron-toolkit/utils": "^4.0.0",
     "@modelcontextprotocol/sdk": "1.11.0",
+    "@napi-rs/keyring": "^1.1.8",
```

Biome (1.9.4)

[error] The key @napi-rs/keyring was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)